image/cli/mascli/functions/provision_fyre

```diff
@@ -79,7 +79,7 @@ function provision_fyre_noninteractive() {
         SIMULATE_AIRGAP_NETWORK=true
         ;;
       -C|--ca-file)
-        REGISTRY_PRIVATE_CA_FILE=$1 && shift
+        REGISTRY_PUBLIC_CA_FILE=$1 && shift
```

Comment:
As with comments in other file .. not sure why we're changing the cluster to use the public registry endpoint?